index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Mobile Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <header class="first">
+        <a class="first-img"><img src="./images/Welcome.png" alt="welcome button at the top left"></a>
+        <div class="first-img" > <img src="./images/Union.png" alt="a union of three vectors"> </div>
```

Comment:
- [ Required ] As requested by a previous reviewer please consider wrapping the `hamburger` menu in the semantic `nav` tag instead of the generic `div` tag. The `nav` tag makes it easier for people using web assistive technologies such as screen readers to identify where the menu is located on the webpage. You can do this using the following markup
```HTML
<nav class="first-img" > <img src="./images/Union.png" alt="a union of three vectors"> </nav>
```